wordpress-mobile/WordPress-iOS
# [Site creation] Add Skip button to Verticals

Fixes #10917 

This turned out to be a bit of a difficult solution, because on the iPhone SE the button would overlap the search field. 

As shown in the branch history, my first solution encapsulated the handling of the button separately from the table view animation, but, in the end, and the offset to be applied to the table view might depend on the position of the Skip button I merged all the logic into the current `TableViewOffsetCoordinator`. Not ideal, but I don't see much better way.

This is how it looks on an iPhone X
![screen recording 2019-01-30 at 4 25 48 pm mov](https://user-images.githubusercontent.com/2722505/51968833-e1ef2880-24ad-11e9-9bad-4f2d2911fb87.gif)

And this is an iPhone SE:
![screen recording 2019-01-30 at 4 23 51 pm mov](https://user-images.githubusercontent.com/2722505/51968839-e7e50980-24ad-11e9-863d-fdd5aa5b2eb3.gif)

Some static screenshots:
iPad Pro 11:
![simulator screen shot - ipad pro 11-inch - 2019-01-30 at 16 33 49](https://user-images.githubusercontent.com/2722505/51968886-fb907000-24ad-11e9-8683-f66d46d8fa85.png)

iPhone SE:
![simulator screen shot - iphone se - 2019-01-30 at 16 29 06](https://user-images.githubusercontent.com/2722505/51968914-0c40e600-24ae-11e9-9727-f6602a65706c.png)
![simulator screen shot - iphone se - 2019-01-30 at 16 29 09](https://user-images.githubusercontent.com/2722505/51968922-106d0380-24ae-11e9-8b5b-3d83faa949ea.png)
![simulator screen shot - iphone se - 2019-01-30 at 16 29 14](https://user-images.githubusercontent.com/2722505/51968923-119e3080-24ae-11e9-8095-d00e6cdfda8a.png)

iPhone X:
![simulator screen shot - iphone xr - 2019-01-30 at 16 26 29](https://user-images.githubusercontent.com/2722505/51968963-27135a80-24ae-11e9-9a8e-c4836b33f9d7.png)
![simulator screen shot - iphone xr - 2019-01-30 at 16 26 32](https://user-images.githubusercontent.com/2722505/51968971-2aa6e180-24ae-11e9-822f-73a965de0884.png)
![simulator screen shot - iphone xr - 2019-01-30 at 16 26 38](https://user-images.githubusercontent.com/2722505/51968987-31cdef80-24ae-11e9-9c60-f07077d0437c.png)

To test:
- Check out the branch, run the tests. 
- Check that the Skip button works as it should

Update release notes:
